src/components/notifications/SMSIR.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div class="mb-3">
+        <label for="smsir-key" class="form-label">{{ $t("API Key") }}</label>
+        <HiddenInput id="smsir-key" v-model="$parent.notification.smsirApiKey" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="smsir-number" class="form-label">{{ $t("Recipient Number") }}</label>
+        <input id="smsir-number" v-model="$parent.notification.smsirNumber" placeholder="9123456789" type="text" maxlength="10" minlength="10" class="form-control" required>
```

Comment:
is the strict 10 char length accurate? Telephone numbers are usually not strictly 10 chars.

Reply:
I defer to your judgenemnt on this one ^^